[libpq] build failure x86-windows x64-windows with features core,icu,nls,openssl,xml,zlib

**Host Environment**
- OS: [Microsoft Windows [Version 10.0.19042.630]]
- Compiler: 
   - Visual Studio 2017 Version: 15.9.29  VS Tools Version: 14.16.27023
   - Visual Studio 2019 Version: 16.8.1  VS Tools Version: 14.28.29333

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone git@github.com:microsoft/vcpkg.git` includes PR #11776 with commit https://github.com/microsoft/vcpkg/commit/3a26eb3dc4d684ca6c1748dccc7a15684b9e8d0a `[intl/gettext/fontconfig] update to native build system #11776` - **it may be relevant, but not sure**
2. `.\bootstrap-vcpkg.bat -verbose -disableMetrics -win64`
3. `set VCPKG_DEFAULT_TRIPLET=x64-windows`
4. `.\vcpkg.exe  install --x-use-aria2 --recurse --keep-going --editable gettext[core] icu[core] libiconv[core] liblzma[core] libpq[core,icu,nls,openssl,xml,zlib] libxml2[core] openssl[core] zlib[core]`

**Failure logs** 
- Cut and past the appropriate build messages from the console output.
>Computing installation plan...
>The following packages will be built and installed:
>    gettext[core]:x64-windows
>    icu[core]:x64-windows
>    libiconv[core]:x64-windows
>    liblzma[core]:x64-windows
>    libpq[core,icu,nls,openssl,xml,zlib]:x64-windows
>    libxml2[core]:x64-windows
>    openssl[core]:x64-windows
>    zlib[core]:x64-windows
>Starting package 1/8: libiconv:x64-windows
>Building package libiconv[core]:x64-windows...
> ...
>Elapsed time for package openssl:x64-windows: 1.308 min
>Starting package 8/8: libpq:x64-windows
>Building package libpq[core,icu,nls,openssl,xml,zlib]:x64-windows...
>-- Downloading postgresql-12.2.tar.bz2...
>-- Extracting source D:/Git/vcpkg_x64/downloads/postgresql-12.2.tar.bz2
>-- Applying patch patches/windows/install.patch
>-- Applying patch patches/windows/win_bison_flex.patch
>-- Applying patch patches/windows/openssl_exe_path.patch
>-- Applying patch patches/windows/Solution.patch
>-- Applying patch patches/windows/MSBuildProject_fix_gendef_perl.patch
>-- Applying patch patches/windows/msgfmt.patch
>-- Applying patch patches/windows/python_lib.patch
>-- Applying patch patches/linux/configure.patch
>-- Applying patch patches/windows/minimize_install.patch
>-- Using source at D:/Git/vcpkg_x64/buildtrees/libpq/src/tgresql-12-3dcd887135
>-- Copying libpq source files to D:/Git/vcpkg_x64/buildtrees/libpq/x64-windows-rel...
>-- Copying libpq source files... done
>-- Applying patch patches/windows/Solution_RELEASE.patch
>-- Applying patch patches/windows/python3_build_RELEASE.patch
>-- Patches applied!
>-- Downloading msys-gettext-0.19.8.1-1-x86_64.pkg.tar.xz...
>-- Downloading msys-libgettextpo-0.19.8.1-1-x86_64.pkg.tar.xz...
>-- Downloading msys-libasprintf-0.19.8.1-1-x86_64.pkg.tar.xz...
>-- Using msys root at D:/Git/vcpkg_x64/downloads/tools/msys2/91955029573d0bcc
>-- Building libpq x64-windows-RELEASE...
>CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:106 (message):
    Command failed: D:/Git/vcpkg_x64/downloads/tools/perl/5.30.0.1/perl/bin/perl.exe build.pl RELEASE libpq
>    Working Directory: D:/Git/vcpkg_x64/buildtrees/libpq/x64-windows-rel/src/tools/msvc 
>    Error code: 1
>    See logs for more information:
>      D:\Git\vcpkg_x64\buildtrees\libpq\build-libpq-x64-windows-RELEASE-out.log
>
>Call Stack (most recent call first):
>  ports/libpq/portfile.cmake:215 (vcpkg_execute_required_process)
>  scripts/ports.cmake:136 (include)
>
>
>Error: Building package libpq:x64-windows failed with: BUILD_FAILED
>Elapsed time for package libpq:x64-windows: 35.99 s
>
>Total elapsed time: 12.87 min
> ...
- Please attach any additional failure logs mentioned in the console output.
[build-libpq-x64-windows-RELEASE-out.log](https://github.com/microsoft/vcpkg/files/5557369/build-libpq-x64-windows-RELEASE-out.log)

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.
build-libpq-x64-windows-RELEASE-out.log Contains the follow error:
>    1>LINK : fatal error LNK1181: cannot open input file 'D:\vcpkg_x64\vs2019\prod\installed\x64-windows\lib\libintl.lib' [D:\vcpkg_x64\vs2019\prod\buildtrees\libpq\x64-windows-rel\libpq.vcxproj]

Note: x86-windows-static and x64-windows-static build fine

--- a/ports/ffmpeg/0007-fix-lib-naming.patch
+++ b/ports/ffmpeg/0007-fix-lib-naming.patch
@@ -8,7 +8,7 @@ index d6c4388..75b96c3 100644
              -lx264)               echo libx264.lib ;;
 +            -lx265)               echo libx265.lib ;;
 +            -lmp3lame)            echo libmp3lame.lib ;;
-+            -liconv)              echo libiconv.lib ;;
++            -liconv)              echo iconv.lib ;;
 +            -lm)                  ;;
              -lstdc++)             ;;
              -l*)                  echo ${flag#-l}.lib ;;

--- a/ports/ffmpeg/CONTROL
+++ b/ports/ffmpeg/CONTROL
@@ -1,6 +1,6 @@
 Source: ffmpeg
 Version: 4.3.1
-Port-Version: 7
+Port-Version: 8
 Homepage: https://ffmpeg.org
 Description: a library to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created.
   FFmpeg is the leading multimedia framework, able to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created. It supports the most obscure ancient formats up to the cutting edge. No matter if they were designed by some standards committee, the community or a corporation. It is also highly portable: FFmpeg compiles, runs, and passes our testing infrastructure FATE across Linux, Mac OS X, Microsoft Windows, the BSDs, Solaris, etc. under a wide variety of build environments, machine architectures, and configurations.

--- a/ports/libpq/CONTROL
+++ b/ports/libpq/CONTROL
@@ -1,6 +1,6 @@
 Source: libpq
 Version: 12.2
-Port-Version: 8
+Port-Version: 9
 Build-Depends: libpq[core,bonjour] (osx)
 Supports: !uwp
 Homepage: https://www.postgresql.org/

--- a/ports/libpq/patches/windows/Solution_Debug.patch
+++ b/ports/libpq/patches/windows/Solution_Debug.patch
@@ -93,7 +93,7 @@ index 51711c2bf..d33fdc3b0 100644
  	{
  		$proj->AddIncludeDir($self->{options}->{nls} . '\include');
 -		$proj->AddLibrary($self->{options}->{nls} . '\lib\libintl.lib');
-+		$proj->AddLibrary($self->{options}->{nls} . '\debug\lib\libintl.lib'); # This must be updated
++		$proj->AddLibrary($self->{options}->{nls} . '\debug\lib\intl.lib'); # This must be updated
  	}
  	if ($self->{options}->{gss})
  	{
@@ -110,7 +110,7 @@ index 51711c2bf..d33fdc3b0 100644
  	{
  		$proj->AddIncludeDir($self->{options}->{iconv} . '\include');
 -		$proj->AddLibrary($self->{options}->{iconv} . '\lib\iconv.lib');
-+		$proj->AddLibrary($self->{options}->{iconv} . '\debug\lib\libiconv.lib'); # This must be updated
++		$proj->AddLibrary($self->{options}->{iconv} . '\debug\lib\iconv.lib'); # This must be updated
  	}
  	if ($self->{options}->{icu})
  	{

--- a/ports/libpq/patches/windows/Solution_Release.patch
+++ b/ports/libpq/patches/windows/Solution_Release.patch
@@ -93,7 +93,7 @@ index 51711c2bf..d33fdc3b0 100644
  	{
  		$proj->AddIncludeDir($self->{options}->{nls} . '\include');
 -		$proj->AddLibrary($self->{options}->{nls} . '\lib\libintl.lib');
-+		$proj->AddLibrary($self->{options}->{nls} . '\lib\libintl.lib'); # This must be updated
++		$proj->AddLibrary($self->{options}->{nls} . '\lib\intl.lib'); # This must be updated
  	}
  	if ($self->{options}->{gss})
  	{
@@ -110,7 +110,7 @@ index 51711c2bf..d33fdc3b0 100644
  	{
  		$proj->AddIncludeDir($self->{options}->{iconv} . '\include');
 -		$proj->AddLibrary($self->{options}->{iconv} . '\lib\iconv.lib');
-+		$proj->AddLibrary($self->{options}->{iconv} . '\lib\libiconv.lib'); # This must be updated
++		$proj->AddLibrary($self->{options}->{iconv} . '\lib\iconv.lib'); # This must be updated
  	}
  	if ($self->{options}->{icu})
  	{

